Justfile

```
@@ -208,13 +201,15 @@ _build-bib $target_image $tag $type $config: (_rootful_load_image target_image t
       --net=host \
       --security-opt label=type:unconfined_t \
       -v $(pwd)/${config}:/config.toml:ro \
-      -v $(pwd)/output:/output \
+      -v $TEMPDIR:/output \
       -v /var/lib/containers/storage:/var/lib/containers/storage \
       "${bib_image}" \
       ${args} \
       "${target_image}:${tag}"
 
-    sudo chown -R $USER:$USER output
+    mkdir -p output
+    sudo mv -f $TEMPDIR "output/${type}"
+    sudo chown -R $USER:$USER "output/${type}"
```

```suggestion
    sudo mv -f $TEMPDIR "output"
    sudo chown -R $USER:$USER "output"
```
We are using the `$TEMPDIR` as basically the output directory, would be best to just move it entirely if possible

---

What if i change it to this?
```
mv -f $TEMPDIR/* output
rmdir $TEMPDIR
chown -R $USER:$USER output
```

That way I we don't delete/overwrite previously built files unless the type is the same. That together with adding `output` to the clean target?